test/integration/voting.test.js

```diff
@@ -0,0 +1,217 @@
+    rank: 3,
+    rate: 3,
+    username: 'iii.element.iii',
+    vote: '3789594637157356',
+  },
+];
+
+const account = {
+  ...accounts.genesis,
+  delegate: {},
+  multisignatures: [],
+  u_multisignatures: [],
+  unconfirmedBalance: '0',
+};
+
+let store;
+let wrapper;
+let listDelegatesApiStub;
+let listAccountDelegatesStub;
+let voteApiStub;
+let accountAPIStub;
+let localStorageStub;
+
+const loginProcess = (votes = []) => {
+  accountAPIStub = stub(accountAPI, 'getAccount');
+  localStorageStub = stub(localStorage, 'getItem');
+  localStorageStub.withArgs('accounts').returns(JSON.stringify([{}, {}]));
+  listDelegatesApiStub = stub(delegateApi, 'listDelegates');
+  listAccountDelegatesStub = stub(delegateApi, 'listAccountDelegates');
+  voteApiStub = stub(delegateApi, 'vote');
+  store = prepareStore({
+    account: accountReducer,
+    peers: peersReducer,
+    voting: votingReducer,
+    transactions: transactionReducer,
+  }, [
+    thunk,
+    accountMiddleware,
+    loginMiddleware,
+    transactionsMiddleware,
+    peerMiddleware,
+  ]);
+
+  accountAPIStub.withArgs(match.any).returnsPromise().resolves({ ...account });
+  store.dispatch(activePeerSet({ network: getNetwork(networks.mainnet.code) }));
+  accountAPIStub.withArgs(match.any).returnsPromise().resolves({ ...account });
+  store.dispatch(accountLoggedIn(account));
+
+  listDelegatesApiStub.returnsPromise()
+    .resolves({ delegates, success: true, totalCount: 20 });
+  listAccountDelegatesStub.returnsPromise()
+    .resolves({ delegates: votes, success: true });
+
+  wrapper = mount(renderWithRouter(Voting, store));
+  expect(store.getState().account).to.be.an('Object');
+  expect(store.getState().voting).to.be.an('Object');
+  expect(store.getState().peers).to.be.an('Object');
+};
+
+const restoreApiMocks = () => {
+  listDelegatesApiStub.restore();
+  listAccountDelegatesStub.restore();
+  voteApiStub.restore();
+  accountAPIStub.restore();
+  localStorageStub.restore();
+};
+
+describe('@integration test of Voting', () => {
+  describe('Scenario: should allow to select delegates in the "Voting" and vote for them', () => {
+    step('I\'m logged in as "genesis"', () => { loginProcess(); });
+
+    step('And next button should be disabled', () => {
+      expect(wrapper.find('button.next').props().disabled).to.be.equal(true);
+    });
+
+    step('When I vote to delegates and next button should be enabled', () => {
+      wrapper.find('.delegate-list input').at(0).simulate('change', { target: { value: 'true' } });
+      wrapper.find('.delegate-list input').at(1).simulate('change', { target: { value: 'true' } });
+      const selectionHeader = wrapper.find('.selection h4').text();
+      expect(selectionHeader).to.be.equal('2');
+      expect(wrapper.find('button.next').props().disabled).to.be.equal(false);
+    });
+
+    step('Then I must be able to go to next step', () => {
+      expect(wrapper.find('button.confirm').exists()).to.be.equal(false);
```

Comment:
This would be better as
`expect(wrapper.find('button.confirm')).to.be.not.present();`
For the same reason as above.
https://github.com/producthunt/chai-enzyme#present